### PR TITLE
Upgrade arcgis module and fix example 8 on ArcGIS Notebooks

### PR DESCRIPTION
**Pull request by clechot**

### CODE REVIEW

File: pyprt/pyprt_arcgis/pyprt_arcgis.py [hunk 1]
```diff
@@ -84,7 +84,10 @@ def arcgis_to_pyprt(feature_set):
                 coord_list = geo.coordinates()
 
                 for face_idx, coord_part in enumerate(coord_list):
-                    in_geo = Geometry({"rings": [coord_part]})
+                    if isinstance(coord_part, np.ndarray):
```
Comment (mistafunk) [inline pyprt/pyprt_arcgis/pyprt_arcgis.py:87]: This coordinates() return type distinction is within a code block `if geo.type == 'Polygon'` - I would expect that each Polygon (subclass of Geometry) has the same coordinates data type? Why exactly is this distinction necessary? When do we hit the else path?

The API doc only mention numpy.ndarray as data type for all Geometry subclasses: https://developers.arcgis.com/python/api-reference/arcgis.geometry.html

**Review by mistafunk — CHANGES_REQUESTED** [review]
See inline comments.

File: pyprt/pyprt_arcgis/pyprt_arcgis.py (hunk 1 above)
Comment (clechot) [reply pyprt/pyprt_arcgis/pyprt_arcgis.py:87]: The "coordinates()" function always returns a numpy array. But the distinction has to be done on the "coord_part" variable. Indeed inn some cases, "coord_part" is an array of arrays and in some other cases, it's an array of lists (in the "test_arcgis_with_holes" test for instance).

Comment (mistafunk) [reply pyprt/pyprt_arcgis/pyprt_arcgis.py:87]: It is unexpected that the coordinates sometimes are a 'array of array' and sometimes 'array of list' (within the same feature class). One explanation could be the presence of holes in a feature. Let's keep an eye on this. Accepted for now.

**Review by mistafunk — APPROVED** [review]

**Pull request merged**